src/sentry/static/sentry/app/components/stackedBarChart.tsx

```diff
@@ -271,14 +318,20 @@ class StackedBarChart extends React.Component {
     );
   };
 
-  getMinHeight(index, _pointLength) {
+  getMinHeight(index: number, _pointLength: number): number {
```

Comment:
```suggestion
  getMinHeight(index: number): number {
```

Reply:
as the second parameter is not being used, I would remove it from the function and here https://github.com/getsentry/sentry/blob/424938bc4d19b0897b87de063541a264e821a2f4/src/sentry/static/sentry/app/components/stackedBarChart.tsx#L344